plugin-src/pactCycleDetector/WizardWidget.ts

```diff
@@ -44,7 +42,8 @@ export class WizardWidget extends PopupWidget {
         const command: string = await renderManager.getValueOf(this.commandInputId)
         let process: NodeChildProcess
         try {
-            process = coreUtil.runShellCommand(command) as NodeChildProcess
+            // todo: pass cwd: getRootFolder().getSrcPath() as option?
+            process = environment.runShellCommand(command, {}) as NodeChildProcess
```

Comment:
when options are optional, they don't have to be specified 😉 :
```suggestion
            process = environment.runShellCommand(command) as NodeChildProcess
```